config/crd/patches/cainjection_in_chyts.yaml

```diff
@@ -3,5 +3,5 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
+    cert-manager.io/inject-ca-from: $(WEBHOOK_CERTIFICATE_NAMESPACE)/$(WEBHOOK_CERTIFICATE_NAME)
```

Comment:
Why is it changed, are we using wrong variables currently?

Reply:
Yes, there was a mistake. That's why previously the annotation looked like:
```
cert-manager.io/inject-ca-from: '{{ .Release.Namespace }}/{{ include "ytop-chart.fullname" . }}-$(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)'
```